docs/DeveloperGuide.adoc

```diff
@@ -297,15 +299,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |project leader |add a project member | asssign tasks to them
+
+|`* * *` |project leader |add tasks |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |project leader |change task status | ger reminder of the progress of each task
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |project leader |sync team members schedules |Find a time slot where the maximum number of people, if not all, can attend
```

Comment:
The capitalisation should be consistent. "find" is capitalise but before that is not